index.html

```diff
@@ -2470,6 +2470,12 @@ <h2 class="section-title">FAQ's</h2>
         <div id="faq-general">
           <ul class="qcFaqList qcScheduleList">
             <li class="toggle">
+              <div>How does Write/Speak/Code define coders?</div>
```

Comment:
Can we get quotes around `coders` in the question, just like in the body?

Other wise 👍 :shipit: Thanks for picking this up!